Makefile

```diff
@@ -40,7 +40,7 @@ DOC_FILENAME	?= oci-image-spec
 
 EPOCH_TEST_COMMIT ?= v0.2.0
 
-default: help
+default: check-license validate-examples lint test
```

Comment:
`validate-examples` is a subset of `test`, so we can drop it from the list of prereqs here.